digital-anonymity.md

```diff
@@ -0,0 +1,66 @@
+| Author | TallAmericano |
+| :---- | :---- |
+| Title | Tools and Strategies for Digital Anonymity |
+| Headline  | Looking at Decentralised Identities and Tools and Strategies for Digital Anonymity. |
+| Cypherpunktimes | link |
+| Publish Date | xx Mar 2024 |
+
+| Social Media | Link |
+| :---- | :---- |
+| Twitter | twitter link |
+| Twitter | thread link… |
+| Twitter | poll link… |
+| Reddit  | link… |
+| Bison Relay | News feed title |
+| Other platforms | Instagram and TikTok |
+
+Thesis statement - This article looks at Tools and Strategies for Digital Anonymity..
```

Comment:
What is your thesis statement for this piece? The structure for this should be something like " what topic you are looking into and why you're conducting this investigation ".

Reply:
Sorry must have missed this. Have updated last week 'Looking at the possibility of Digital Anonymity and if it's achievable.'